recipes/trata/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 16 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "trata" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/LLNL/trata/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 6dc8ce77a28efaa9388723b8afd62e2d5234a81b2ac5323e71c4fa3c17e1837d
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # More info about selectors can be found in the conda-build docs: 
+  # https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

Comment:
This one can probably be noarch.

```suggestion
    noarch: python
    number: 0
```

Reply:
It does look like noarch. Can you please confirm?

Reply:
no arch didn't work

Reply:
What failed?

Reply:
@ocefpaf please see: https://github.com/conda-forge/staged-recipes/runs/21926254364

Reply:
We can't. Azure doesn't keep the logs long enough. Can you please describe it or try it again?

![Screenshot from 2024-03-27 15-11-27](https://github.com/conda-forge/staged-recipes/assets/950575/10900e01-1956-449c-a046-b9c5da37c8b6)


PS: It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.